src/style.css

```diff
@@ -1 +1,91 @@
 /* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+  /* height: 60px; */
+  align-items: center;
+  position: sticky;
```

Comment:
No need it position sticky

Reply:
delete it and add position: relative to .nav__item